File: backend/DataAccess/DaoGenerated/システム共有/tt_afjutogai_dokujisystemDto.cs
// *******************************************************************
// 業務名称　: 健康管理システム
// 機能概要　: DTO定義
//             住登外者独自施策システム等情報テーブル
// 作成日　　: 2023.01.23
// 作成者　　: 屠
// 変更履歴　:
// *******************************************************************
namespace BCC.Affect.DataAccess
{
    public class tt_afjutogai_dokujisystemDto : DaEntityModelBase
    {
		public const string TABLE_NAME = "tt_afjutogai_dokujisystem";
		public string atenano { get; set; }                       //宛名番号
		public int rirekino { get; set; }                         //履歴番号
		public string dokujisystemid { get; set; }                //独自施策システム等ID
    }
}

File: backend/BusinessService/Db/DaoGenerated/システム共有/tt_afjutogai_dokujisystemDto.vb
' *******************************************************************
' 業務名称　: 互助防疫システム
' 機能概要　: DTO定義
'             住登外者独自施策システム等情報テーブル
' 作成日　　: 2024.07.23
' 作成者　　: 
' 変更履歴　:
' *******************************************************************
Namespace Jbd.Gjs.Db
    Public Class tt_afjutogai_dokujisystemDto
        Inherits DaEntityModelBase
        Public Const TABLE_NAME As String = "tt_afjutogai_dokujisystem"
        Public Property atenano As String                       '宛名番号
        Public Property rirekino As Integer                         '履歴番号
        Public Property dokujisystemid As String                '独自施策システム等ID
    End Class
End Namespace
